internal/cmd/provider/flags.go

```diff
@@ -0,0 +1,28 @@
+package provider
+
+import "github.com/urfave/cli/v2"
+
+var flagProviderType = &cli.StringFlag{
+	Name:     "type",
+	Usage:    "[Required] Type of the provider",
+	Required: true,
+}
+
+var flagProviderVersionProtocols = &cli.StringSliceFlag{
+	Name:  "protocols",
+	Usage: "Protocols supported by the provider",
+	Value: cli.NewStringSlice("5.0"),
+}
+
+var flagGoReleaserDir = &cli.StringFlag{
+	Name:  "goreleaser-dir",
+	Usage: "Directory containing the GoReleaser build artifacts",
+	Value: "dist",
+}
+
+var flagSigningKeyID = &cli.StringFlag{
+	Name:     "siging-key-id",
+	Usage:    "ID of the signing key used to sign the provider version",
+	EnvVars:  []string{"GPG_FINGERPRINT"},
```

Comment:
Re: 1
That's because this is the environment variable that GoRelease uses for signing. If you run it in a single CI/CD pipeline (which most folks probably will), the will be no need for setting this explicitly.

Re: 2
I'll rename the flag to match the environment variable (`gpg-fingerprint`)